test/integration/options/forbidOnly.spec.js

```diff
@@ -26,13 +26,22 @@ describe('--forbid-only', function() {
 
   it('should fail if there are tests marked only', function(done) {
     var fixture = path.join('options', 'forbid-only', 'only');
-    runMochaJSON(fixture, args, function(err, res) {
-      if (err) {
-        return done(err);
-      }
-      expect(res, 'to have failed with error', onlyErrorMessage);
-      done();
-    });
+    var spawnOpts = {stdio: 'pipe'};
+    runMocha(
+      fixture,
+      args,
+      function(err, res) {
+        if (err) {
+          return done(err);
+        }
+        expect(res, 'to satisfy', {
```

Comment:
I _believe_ you can use one of our custom assertions here, but I could be wrong

```js
expect(res, 'to have failed with output', new RegExp(onlyErrorMessage));
```

this should check for exit code > 1 and match on `STDOUT` of the child process.

Reply:
Works!